runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -491,6 +491,18 @@ public String getUrl() {
     }
 
     /**
+     * Get the content height of current web page/app.
+     * NOTE: When page finished loading, it's content size info may not be updated.
```

Comment:
No, this means when finished loading. it is possible that the value is still not be updated. but if it happens, just a little while(i test waiting 50 ms). it will be updated.